_posts/2023/05/2023-05-02-release-notes.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+date: 2023-05-02
+title: "v0.29.0: Release Notes"
+author: olenagerasimova
+description: |
+  Key features and improvements description in the recent
+  version v0.30.0, released just a few days ago.
+keywords:
+  - artipie
+  - binary artifacts
+  - hosting of artifacts
+  - Abstract storage
+  - Security policy
+  - Permissions
+  - Filtering
+  - Rest API
+---
+
+Artipie new version [v0.30.0](https://github.com/artipie/artipie/releases/tag/v0.30.0) was released on May 2. 
+In the previous month we added several features, here are the main three of them:
+
+- Artifacts filtering provides means to filter out resources of a repository by specifying patterns of resource location. 
```

Comment:
```suggestion
- Artifacts filtering provides a means to filter out resources of a repository by specifying patterns of resource location. 
```

Reply:
that's defiantly not correct as word "means" is plural and supposition "a" can be used only with singular